frontend/src/libs/components/chat-sidebar/chat-sidebar.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useSearch } from '#libs/hooks/hooks.js';
+
+import { Card, Icon, Search } from '../components.js';
+import styles from './styles.module.scss';
+
+const mockedChats = [
+  { id: 1, name: 'Relationship' },
+  { id: 2, name: 'Friends' },
+  { id: 3, name: 'Family' },
+  { id: 4, name: 'Work' },
+];
+
+const mockedSelectedChat = {
+  id: 1,
+};
+
+const ChatSidebar: React.FC = () => {
+  const { filteredElements, setFilter } = useSearch(mockedChats, 'name');
```

Comment:
Please check mobile part for this implementation

Reply:
@v0ldemar01 what do exactly do you mean? Hook implementation or usage?

Reply:
'name' => 'title' at least

Reply:
@v0ldemar01 but in db schema there name in chat topics 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/7f0bc96c-4b68-433d-b809-018ddaf0dfbb)

Reply:
Got it